src/xyObject/xyObjectMinMaxValues.ts

```diff
@@ -0,0 +1,31 @@
+import type { Point } from '../types';
+
+import { xyObjectCheck } from './xyObjectCheck';
+
+/**
+ * Returns all min and max values of an array of points.
+ * @param points - Array of points {x,y}.
+ * @returns - Object with the 4 extrema.
+ */
+export function xyObjectMinMaxValues(points: Point[]): {
+  minX: number;
+  maxX: number;
+  minY: number;
+  maxY: number;
+} {
```

Comment:
@lpatiny Shape of the return looks good to you? The `x` variant returns `{min,max}`.